### ISSUE TITLE
TvOS Feature Request: Switchtimer

### ISSUE THREAD

**Issue by tonyschuite** [zipleen/tvheadend-iphone-client#228]
I would like to be able to set a timer for a specific show (not a recording).
preferably to make it switch to that channel at the start of the show.
preferably from outside the app (ie watching netflix and make it switch to tvhclient and that specific channel).

**Comment by zipleen** [comment]
hi!
what do you mean by setting a timer for a specific show? recording based on time?

- and are you talking about the tvOS version?

- switching to the channel at the start of the show would be doable, if you were inside of the app already - I would need to figure out how to present this option to the user and code everything around it

- the feature working outside of the app is going to be impossible, apple does not allow that. in iOS I could have a notification saying that the program is about to start and clicking on that notification would start tvhclient and connect. But for tvOS there's no notifications at all, apple does not allow that.

**Comment by tonyschuite** [comment]
Yes, I am talking about the tvOS version. but I do think it would be nice for iOS as well.

so to elaborate:
I would like to be able to set a timer when clicking on a show in the guide (button in program info?). This timer would at least notify me of the program starting, but preferably would switch the channel when the program starts. 

I was afraid of this not being possible for the tvOS version.
However I did read up on it a little and in tvOS 11 you should be able to access some kind of notification.

https://stackoverflow.com/questions/43139057/apple-push-notifications-in-tvos
especially the link to the conference video is interesting.

**Comment by tonyschuite** [comment]
oh, and this is because I used the switchtimer addon in Kodi. Switched to ATV so no Kodi
